src/posts/fsweekend-2024.md

```diff
@@ -0,0 +1,106 @@
+
+Our team was very delighted to be talking to all of you that came to visit us. It is amazing to feel that appreciation and support from the community!
+
+## A380X Trailer Reveal
+
+This year, a new concept was introduced by the FS Weekend organizers, where the theatre was managed by [FSElite](https://fselite.net/) and where exhibitors could give a presentation or showcase about their products or latest developments. Of course, we jumped on the opportunity to create a new trailer of the A380X to be released and revealed at FS Weekend.
+
+[![Packed room in anticipation of our presentation](/img/notam-images/fsweekend2024/packed-room.png)](/img/notam-images/fsweekend2024/packed-room.png)
+
+On Saturday, we were faced with a packed room of people eagerly awaiting our presentation so we jumped straight in with first talking about who we are and what it means to be a group of volunteers working purely out of interest and passion on Free and Open Source projects in the flight simulation world.
+
+After that we played our new A380X trailer for the first time, while it was also being streamed live on the FSElite Youtube channel, as well as being released on our own Youtube channel, where you can watch it in its full glory.
+
+In the style of "One more thing", we decided to show off the work that has gone in our cabin in the final 30 seconds of the video as a surprise. Stay tuned for more!
+
+<iframe width="860" height="500" src="https://www.youtube.com/embed/nyqyL8QcAhs?si=HWekuQAZ1QQJJeJf" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen style="width:860px; height:500px;"></iframe>
+
+## A32NX Live VATSIM Flight
+
+For the second time in a row, our team set up a live flight with the A32NX on VATSIM. Both to promote our beautiful A32NX project, as well as promote VATSIM as an online ATC experience that enhances the realism of flight simulation.
+
+The live flight took us from Groningen airport (EHGG) to Rotterdam (EHRD) in the Netherlands and we worked with the [Dutch division of VATSIM](https://www.dutchvacc.nl/) to have full coverage of our flight.
+
+While two team members of FlyByWire Simulations were flying and communicating with ATC, one person from VATSIM and another of our crew members gave live commentary on the flight. This commentary explained how VATSIM worked and how to get started as a new pilot, or as a new controller.
+
+[![A32NX Live VATSIM Flight Commentary](/img/notam-images/fsweekend2024/live-flight-commentary.png)](/img/notam-images/fsweekend2024/live-flight-commentary.png)
+
+## What's Next?
+
+As FS Weekend 2024, our team started packing up and immediately started to think about the next event where we want to show you all the amazing things we do.
```

Comment:
```suggestion
As FS Weekend 2024 came to a close, our team started packing up and immediately started to think about the next event where we want to show you all the amazing things we do.
```

Seems like you trailed off thought here, added a small contextual note.

Reply:
And FS Weekend -> FSWeekend